configuration/structure.json

```diff
@@ -2501,10 +2501,10 @@
                     "friendlyName": "Contribute to Babylon.js",
                     "content": "contribute/toBabylon",
                     "children": {
-                        "Prerequisits": {
-                            "friendlyName": "Prerequisits for Contributing",
+                        "Prerequisites": {
```

Comment:
I think we can't make this change, else it will break existing links pointing to this page.

cc @RaananW

Reply:
It seems that there are no other references to this page inside the documentation:

![image](https://user-images.githubusercontent.com/115980073/215283442-7d8e0bd4-dac1-47b2-9dc6-80f78d35d622.png)

Nor in the forums (if that's what you were thinking about):

![image](https://user-images.githubusercontent.com/115980073/215282501-9b096b90-6948-48ba-a561-39946ba1b252.png)

Even Google just has this one result:

![image](https://user-images.githubusercontent.com/115980073/215282559-eabe4e76-8a8f-408d-9275-d8d769e5dd34.png)

It does seem there's a `sitemap.xml` that I missed (will fix in a separate commit), but nothing else on GitHub either:

![image](https://user-images.githubusercontent.com/115980073/215283215-d0ef194f-f03a-46f8-a4af-060214385b7a.png)

Reply:
I was not thinking of references inside the documentation but references that could have been used/shared by users, either as bookmarks or inside some docs for eg (or sent by emails, ...).

Reply:
Gotcha. Though I doubt anyone would bookmark [this minimalistic page](https://doc.babylonjs.com/contribute/toBabylon/Prerequisits), it's of course your decision :)

Perhaps it's possible to set up an alias somehow so that bookmark visitors will be redirected?

Reply:
There is a redirection json file that you can use to redirect from the old URL to the new one. Would be great if you add it there!
it will require an approval from an admin (because it is a configuration json), but that shouldn't be a problem.

Reply:
I think I found the right file, but I guess the redirects don't work locally. so a deployment on Vercel is the only way to test?

Reply:
yes, these are server-side redirects. LGTM! thanks a lot